README.md

```diff
@@ -8,13 +8,52 @@ Introducing the nostr-java library, a solution written in java for generating, s
 - Java 19+
 
 ## Usage
-To use the library in your project, add the following dependency to your pom.xml file:
+To use the library in your project, add the following dependency to your pom.xml file.
+
+Define the value for the most recent version available:
+```xml
+    <properties>
+        <nostr-java.version>0.6.1-SNAPSHOT</nostr-java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
property:
  `<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>` 
is already defined in [nostr-java's pom.xml](https://github.com/tcheeric/nostr-java/blob/71c1c49b6480778cc9a445df691a55adc65aabe2/pom.xml#L33).

your project might have additional explicit dependency for it above and beyond what's required for nostr-java, but that is exclusive to your project and unnecessary here as part of general instruction.